### PR TITLE
Thomas/changes

### PR DESCRIPTION
**Pull request by ThomasVy**
I updated some files and the read me. Also added database stuff to prepare for lap

### CODE REVIEW

File: server/migrate.psql [hunk 1]
```diff
@@ -252,6 +252,7 @@ CREATE TABLE public.packet (
     allowcharge boolean,
     contractorerror boolean,
     highvoltageenable boolean
+    lap boolean
```
Comment (bill-luu) [inline server/migrate.psql]: I think you need a comma on the line above

File: server/README.adoc [hunk 2]
```diff
@@ -3,7 +3,8 @@
 Express.js server running in node
 
 == Build
-`npm run start` to run dev server. Can be hit at `http://localhost:4000/`.
+'npm install' to install dependencies. Run this first.
```
Comment (bill-luu) [inline server/README.adoc]: "Run this first" is redundant

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+server/config.js
```
Comment (bill-luu) [inline .gitignore]: Good call!

File: server/migrate.psql (hunk 1 above)
Comment (ThomasVy) [reply server/migrate.psql]: Got you

File: server/README.adoc (hunk 2 above)
Comment (ThomasVy) [reply server/README.adoc]: I love you <3

File: .gitignore (hunk 3 above)
Comment (ThomasVy) [reply .gitignore]: You're a good call <3

**Pull request merged**